<launch>
  <arg name="l" default="0.04"/>
  <arg name="ci" default="1"/>

  <node name="pose_publisher" pkg="aruco_markers" type="pose_estimation" args="-l=$(arg l) -ci=$(arg ci)" required="true"/>
</launch>